libpq/meta.yaml

```diff
@@ -0,0 +1,17 @@
+package:
+  name: libpq
+  version: 9.4.5
+
+build:
+  number: 1
+
+source:
+  fn: postgresql-9.4.5.tar.bz2
```

Comment:
Why do you call `postgresql` libpq?  Will the default channel version suffice?

http://anaconda.org/anaconda/postgresql